src/data/actions/assets.test.jsx

```diff
@@ -159,15 +206,86 @@ describe('Assets Action Creators', () => {
     expect(store.dispatch(actionCreators.uploadExceedMaxCount(1))).toEqual(expectedAction);
   });
   it('returns expected state from uploadAssetFailure', () => {
-    const expectedAction = { file: 'file', response: 'response', type: assetActions.UPLOAD_ASSET_FAILURE };
+    const expectedAction = { asset: 'file', response: 'response', type: assetActions.UPLOAD_ASSET_FAILURE };
```

Comment:
Just changing the test to match the changes I made to the uploadAssetFailure action creator.